development/shellcheck.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# lint all *.sh files
+shellcheck --version && find . -type f -name '*.sh' ! -path './node_modules/*' -print0 | xargs -0 shellcheck
```

Comment:
Could you move `shellcheck --version` to a separate line? Now that it's been moved to a script, there's no reason to do that in one line like this.